script.js

```diff
@@ -11,4 +13,328 @@ hamburger.addEventListener('click', toggleMenu);
+    language.innerHTML = projects[i].languages[j];
+    languages.appendChild(language);
+  }
+  project.appendChild(languages);
+
+  const button = document.createElement('button');
+  button.setAttribute('id', 'project-button');
+  button.classList.add('button');
+  button.innerHTML = projects[i].buttonName;
+  project.append(button);
+}
+
+function createPopup(index) {
+  const project = projects[index];
+  const x = document.querySelector('.works');
+
+  const overlay = document.createElement('div');
+  overlay.classList.add('overlay');
+  x.appendChild(overlay);
+
+  const popupWindow = document.createElement('div');
+  popupWindow.setAttribute('id', 'popup-window');
+  popupWindow.classList.add('popup-section');
+
+  const popupCloseButton = document.createElement('button');
+  popupCloseButton.setAttribute('id', 'close-button');
+  popupCloseButton.classList.add('close-button');
+  popupCloseButton.innerHTML = '&times';
+  popupWindow.appendChild(popupCloseButton);
+
+  const popupImageDesk = document.createElement('img');
+  popupImageDesk.setAttribute('src', project.popupImageDesktop);
+  popupImageDesk.classList.add('desktop', 'popup-image');
+  popupWindow.appendChild(popupImageDesk);
+
+  const popupImageMob = document.createElement('img');
+  popupImageMob.setAttribute('src', project.popupImageMobile);
+  popupImageMob.classList.add('mobile', 'popup-image');
+  popupWindow.appendChild(popupImageMob);
+
+  const popupTitleButtonsContainer = document.createElement('div');
+  popupTitleButtonsContainer.classList.add('title-buttons');
+  popupWindow.appendChild(popupTitleButtonsContainer);
+
+  const popupTitle = document.createElement('h3');
+  popupTitle.classList.add('popup-title');
+  popupTitle.innerHTML = project.popupTitle;
+  popupTitleButtonsContainer.appendChild(popupTitle);
+
+  const popupButtonsContainerDesktop = document.createElement('div');
+  popupButtonsContainerDesktop.classList.add('desktop', 'popup-buttons');
+  popupTitleButtonsContainer.appendChild(popupButtonsContainerDesktop);
+
+  const popupLiveLinkDesk = document.createElement('button');
+  popupLiveLinkDesk.classList.add('button');
+  popupLiveLinkDesk.innerHTML = 'See Live';
+  popupButtonsContainerDesktop.appendChild(popupLiveLinkDesk);
+
+  const popupLiveLinkIconDesk = document.createElement('img');
+  popupLiveLinkIconDesk.setAttribute('src', 'images/see-live-icon.png');
+  popupLiveLinkDesk.appendChild(popupLiveLinkIconDesk);
+
+  const popupSourceLinkDesk = document.createElement('button');
+  popupSourceLinkDesk.classList.add('button');
+  popupSourceLinkDesk.innerHTML = 'See Source';
+  popupButtonsContainerDesktop.appendChild(popupSourceLinkDesk);
+
+  const popupSourceLinkIconDesk = document.createElement('img');
+  popupSourceLinkIconDesk.setAttribute('src', 'images/See-source.png');
+  popupSourceLinkIconDesk.setAttribute('alt', 'Source Link Icon');
+  popupSourceLinkDesk.appendChild(popupSourceLinkIconDesk);
```

Comment:
👏 Thank you for addressing the previous reviewer's feedback and adjusting the button width and height. However, there are still a couple of refinements needed to align the buttons with the design requirements:

- [ ] First, the padding on the top and bottom of the buttons matches the design at 12px. However, the padding on the left and right sides is currently set to 8px, whereas it should be 16px as specified in the design. To ensure consistency, please update your code to match the 16px padding as per the design. This adjustment will help achieve the desired button width.

- [ ] Second, the spacing between the icon and text within the buttons is currently set to `0.4rem`, which is equivalent to `6.4px`. However, according to the design, this gap should be `12px`. To align with the design, please adjust the gap between the icon and text to `12px`. This change will contribute to achieving the intended button width and layout.

![image](https://github.com/danielf-canon/Portfolio/assets/55840999/a3ba6673-cead-44a9-8837-fc3371e86293)

By implementing these adjustments, you'll be able to match the button width and spacing as outlined in the design, resulting in a more polished and consistent appearance. Keep up the excellent work, and happy coding! 🚀💻🌟